scout/server/blueprints/cases/controllers.py

```diff
@@ -624,6 +639,7 @@ def case_report_variants(store: MongoAdapter, case_obj: dict, institute_obj: dic
                 continue
             if case_key == "partial_causatives":
                 var_obj["phenotypes"] = case_obj["partial_causatives"][var_id]
+            add_bayesian_acmg_classification(var_obj)
```

Comment:
@dnil this is the actual fix: notice that the variants that didn't show the bayesian classif were pinned or causative

Reply:
Ah! Well spotted!